### ISSUE TITLE
Async code fails when running on the server.

### ISSUE THREAD

**Issue by onyxring** [onyxring/Roll20Async#2]
Reported in the forums by Scott C.  In correspondence, the following snippets describe:

"Any API that is going to trigger sheetworkers will error out because the self object that the async relies on doesn't exist in the API sandbox."

and

"The methods you need to reset the active character don't exist on the server. There's no way to maintain the sheetworker/active sheet connection."

**Comment by onyxring** [comment]
I confirmed this behavior.  

In some circumstances, API code actually executes sheetworker events on the **server**, rather than in the **browser** (e.g. via the "setWithWorker" function).  In such cases, the asynchronous calls fail when calling the setActiveCharacterId() function, which leverages the sheetworker's event processing pipeline.  The browser version of the event processor differs from the server version, so the events running on the server fail.  

I addressed this with a conditional, inside of setActiveCharacterId(), which detects if the code is running on the server or in the browser, and leverages the appropriate event processor.

I've communicated this with the original poster and will close this on confirmation that the behavior is addressed.
